homeassistant/components/media_player/__init__.py

```diff
@@ -89,14 +90,14 @@ def volume_down(hass, entity_id=None):
 
 
 def volume_mute(hass, entity_id=None):
```

Comment:
If we expect the service data to contain a mute boolean, we should add that to this method signature.